src/pages/editor.tsx

```diff
@@ -1,25 +1,105 @@
 import React from "react";
-import styled from "styled-components";
+import styled, { AnyStyledComponent } from "styled-components";
 import { Navbar } from "../components/navbar";
 import "../components/app/normalize.css";
 
-import { background, textPrimary, grey } from "../constants";
+import { background, grey } from "../constants";
 
-const StyledEditor = styled.div`
+const StyledEditor: AnyStyledComponent = styled.div`
   height: 100vh;
   width: 100vw;
   display: flex;
-  flex-direction: row;
+  flex-direction: column;
   justify-items: stretch;
   background: ${background};
   overflow: auto;
 `;
 
+const MainWrapper: AnyStyledComponent = styled.main`
+  display: flex;
+  align-items: flex-start;
+  flex-direction: row;
+  flex-wrap: none;
+  flex: 1;
+`;
+
+const StyledHeader: AnyStyledComponent = styled.div`
+  height: 72px;
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  justify-items: stretch;
+  align-items: center;
+  justify-content: center;
+  color: white;
+  border: 1px solid ${grey};
+  border-left: none;
+`;
```

Comment:
```suggestion
border-bottom: none;
`;
```
The border underneath the header is thicker than the other borders because there's actually 2 borders: one from the header and one from the `StyledTitle`. I think removing the bottom border from the header is the easiest way to resolve this.